packages/office-ui-fabric-react/src/components/Link/Link.scss

```diff
@@ -42,6 +41,11 @@
   &:active {
     color: $ms-color-themePrimary;
   }
+
+  &:focus {
```

Comment:
Done. I've added a temporary `focus-outline()` mixin. I've opened an [issue for Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core/issues/892) to get the mixin added into Core, which would then allow us to remove it from React.